iconpacks/README.md

```diff
@@ -0,0 +1,25 @@
+# Eclipse Icon Packs
+
+This directory contains community-driven icon packs for the Eclipse IDE.
+These icon packs provide refreshed, high-quality vector icons to improve the look and feel of Eclipse.
+
+## Available Icon Packs
+
+### Eclipse Dual Tone
+A modern dual-tone icon pack with clean, scalable vector icons for Eclipse IDE components.
+
+**Location:** `eclipse-dual-tone/`
+
+See the individual icon pack directories for more details on contributing and using each pack.
+
+## Contributing
+
+Each icon pack has its own style guide and contribution process.
+Please refer to the README in each icon pack's directory for specific guidelines.
+
+## Purpose
```

Comment:
This section desribes why this PR is necessary. I don't think it is needed to justify the existence of this directory once the PR is merged.